tests/app/test_dependencies.py

```diff
@@ -38,6 +44,18 @@
     LiteratureSearchTool,
     MorphologyFeatureTool,
 )
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session
+
+
+def test_get_settings(monkeypatch):
```

Comment:
Can you not use the ` patch_required_env` from `conftest.py` ?

Reply:
I included it in each test where it makes sense.